src/features/unified-share-modal/__tests__/ContactsField-test.js

```diff
@@ -31,20 +31,23 @@ describe('features/unified-share-modal/ContactsField', () => {
             email: 'x@example.com',
             id: '12345',
             text: 'X User',
+            name: 'X User',
```

Comment:
Hm, do we need both `name` and `text`? seems that `text` was already there, and is always the same as name.

Reply:
I don't actually know much about the API response and what the type is used by. I am mostly just adapting to what exists here, might merit a deeper look though.

Reply:
I share @booc0mtaco's question and think it should be at least investigated before moving forward.